src/pages/RestaurantFrontPage.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "../styles.css";
+
+import RestaurantNameList from "../components/RestaurantNameList";
+
+class Restaurant extends React.Component {
+  state = {
+    restaurantNameList: []
+  };
+
+  loadData = async () => {
+    const response = await fetch(
+      "https://cors-anywhere.herokuapp.com/https://test-places-response.s3.eu-west-3.amazonaws.com/response.json"
+    );
+
+    const data = await response.json();
+    this.setState({
+      restaurantNameList: data.results
+    });
+    console.log(data.results);
```

Comment:
Also, we should remove `console.log`s from the code, when we are done with testing our code. You can remove this now.